packages/jest-circus/src/index.ts

```diff
@@ -145,7 +146,7 @@ const test: Global.It = (() => {
       fn?: Circus.TestFn,
       timeout?: number,
     ): void => _addTest(testName, mode, concurrent, fn, failing, timeout, true);
-    return failing;
+    return failing as Failing;
```

Comment:
Not super happy about this, however it was the only way to stop TS from complaining,